fastlane/lib/fastlane/runner.rb

```diff
@@ -43,6 +46,10 @@ def execute(lane, platform = nil, parameters = nil)
       parameters ||= {}
       begin
         Dir.chdir(path_to_use) do # the file is located in the fastlane folder
+          environment_variables[current_platform] ||= {}
+          environment_variables[current_platform].each do |name, key|
+            ENV[name] = key
```

Comment:
Let's also `UI.verbose` the `name` and `key` so that in verbose mode the user can see that the environment variable is being set 👍